src/matrix/kernelkroneckermat.jl

```diff
@@ -27,31 +27,39 @@ end
 """
 @inline iskroncompatible(κ::Kernel) = false # Default return for kernels
 
-function _kernelmatrix_kroneckerjl_helper(::MOInputIsotopicByFeatures, Kfeatures, Koutputs)
+function _kernelmatrix_kroneckerjl_helper(
+    ::Type{<:MOInputIsotopicByFeatures}, Kfeatures, Koutputs
+)
     return Kronecker.kronecker(Kfeatures, Koutputs)
 end
 
-function _kernelmatrix_kroneckerjl_helper(::MOInputIsotopicByOutputs, Kfeatures, Koutputs)
+function _kernelmatrix_kroneckerjl_helper(
+    ::Type{<:MOInputIsotopicByOutputs}, Kfeatures, Koutputs
+)
     return Kronecker.kronecker(Koutputs, Kfeatures)
 end
 
+"""
+    kronecker_kernelmatrix(k::Union{IndependentMOKernel,IntrinsicCoregionMOKernel}, x::MOI, y::MOI) 
+    where {MOI<:IsotopicMOInputsUnion}
+
+Requires Kronecker.jl: Computes the `kernelmatrix` for the `IndependentMOKernel` and the `IntrinsicCoregionMOKernel`, but returns a lazy kronecker product. This object can be very efficiently inverted or decomposed. See also [`kernelmatrix`](@ref). 
```

Comment:
I have no idea why the formatter didn't complain about the line length here. Looks to me like it should be something like
```suggestion
Requires Kronecker.jl: Computes the `kernelmatrix` for the `IndependentMOKernel` and the
`IntrinsicCoregionMOKernel`, but returns a lazy kronecker product. This object can be very
efficiently inverted or decomposed. See also [`kernelmatrix`](@ref).
```